pkg/util/pointer/pointer.go

```diff
@@ -60,3 +60,9 @@ func Int32PtrDerefOr(ptr *int32, def int32) int32 {
 	}
 	return def
 }
+
+// BoolPtr returns a pointer to a bool
+func BoolPtr(b bool) *bool {
```

Comment:
?

Reply:
This is used in https://github.com/kubernetes/kubernetes/pull/55803/files#diff-5b39aa0270237ea464fd6c9746548aa2R158

`AllowPrivileged` is a pointer.

```
+	if obj.BaseConfig.AllowPrivileged == nil {
+		obj.BaseConfig.AllowPrivileged = utilpointer.BoolPtr(true)
+	}
```

Reply:
Indeed, thanks!